### ISSUE TITLE
Drush 10.3.4 think it is version 9

### ISSUE THREAD

**Issue by macmladen** [drush-ops/drush#4568]
**Describe the bug**
Not sure how to declare this but I was puzzled if wrong `drush` came to me.

**To Reproduce**
I wanted to do **site install** from existing configuration but had some issues so I turned on **debug** and it said **VERSION: 9** while I knew it was 10.

```
mladen@Fish 09:31:45 $ fin drush --version
Drush Launcher Version: 0.6.0
Drush Commandline Tool 10.3.4
```
```
~/Sites/
mladen@Fish 14:44:11 $ fin drush --version --vvv --debug
Drush Launcher Version: 0.6.0
ROOT: /var/www
DRUSH VERSION: 9
DRUPAL ROOT: /var/www/web
COMPOSER ROOT: /var/www
VENDOR ROOT: /var/www/vendor
 [preflight] Config paths: /var/www/vendor/drush/drush/drush.yml
 [preflight] Alias paths: /var/www/web/drush/sites,/var/www/drush/sites
 [preflight] Commandfile search paths: /var/www/vendor/drush/drush/src
Drush Commandline Tool 10.3.4
~/Sites/
```

**Expected behavior**
What did you expect would happen?

`DRUSH VERSION: 10`

**Actual behavior**
What happened instead?

`DRUSH VERSION: 9`

**Workaround**
Is there another way to do the desired action?

### System Configuration
| Q               | A
| --------------- | ---
| Drush version?  | 10.3.4
| Drupal version? | 9.0.6
| PHP version     | 7.4
| OS?             | Linux/Docksal

**Additional information**

**Comment by greg-1-anderson** [comment]
Check to see if you have a site-local Drush:

`composer info drush/drush`

**Comment by macmladen** [comment]
I do, I added it and that can be seen in the first block, but here it is explicitly:

```
~/Sites/
mladen@Fish 14:44:26 $ fin composer info drush/drush
name     : drush/drush
descrip. : Drush is a command line shell and scripting interface for Drupal, a veritable Swiss Army knife designed to make life easier for those of us who spend some of our working hours hacking away at the command prompt.
keywords : 
versions : * 10.3.4
type     : library
license  : GNU General Public License v2.0 or later (GPL-2.0-or-later) (OSI approved) https://spdx.org/licenses/GPL-2.0-or-later.html#licenseText
homepage : http://www.drush.org
source   : [git] https://github.com/drush-ops/drush.git 802419a6eb0232bdb977788e6a828cd4fa5bdb11
dist     : [zip] https://api.github.com/repos/drush-ops/drush/zipball/802419a6eb0232bdb977788e6a828cd4fa5bdb11 802419a6eb0232bdb977788e6a828cd4fa5bdb11
path     : /var/www/vendor/drush/drush
names    : drush/drush
```

**Comment by weitzman** [comment]
I suspect that drush-launcher is just misrepresenting the version it sees. Are you having any actual problems or is this version report the only issue? If there is nothing more, I'd like to close this and release a new version of drush-launcher for folks to use. Last release was in 2018.

**Comment by macmladen** [comment]
I had a different type of a problem and only in **--debug** did I see that so it puzzled me **if** the right version (local to project and that is 10.3.4) was used in order to trace problems with installing from existing configuration export.

Everything went right (from `drush` perspective), **site:install** was performed but I was wondering if **launcher** picked the wrong version somehow and that it was making an issue.

So probably it is launcher detection that is wrong and visible only with **--debug** but it may puzzle someone else, hence the issue.

**Comment by weitzman** [comment]
Looks like this was fixed in https://github.com/drush-ops/drush-launcher/pull/76 so I released 0.7.0 of drush-launcher https://github.com/drush-ops/drush-launcher/releases/tag/0.7.0
